xml/System.Threading.Tasks/TaskScheduler.xml

```diff
@@ -93,11 +93,11 @@
   
  To create the example, create a WPF application project in Visual Studio and assign it a name of your choice. Then do the following:  
```

Comment:
Name of your choice is not really right here given the code above is assuming a namespace so if you copy and paste this code it won't work if you give a random name. So maybe we should tell folks to create a project with a specific name?